sdk/python/kfp/deprecated/cli/cli.py

```diff
@@ -27,10 +27,8 @@
 from kfp.deprecated.cli.output import OutputFormat
 from kfp.deprecated.cli import components
 
-_NO_CLIENT_COMMANDS = [
-    'diagnose_me',
-    'components'
-]
+_NO_CLIENT_COMMANDS = ['diagnose_me', 'components']
+
```

Comment:
nit: looks like this diff is still here. Perhaps editor auto-format?

Reply:
Oops I will check that again!

Reply:
It should be fixed now -- the file was not included in the last commit.